src/frontend/src/lib/utils/nav.utils.ts

```diff
@@ -71,7 +71,7 @@ export const loadRouteParams = ($event: LoadEvent): RouteParams => {
 	const token = searchParams?.get('token');
 
 	const replaceEmoji = (input: string | null): string | null => {
-		if (input === null) {
+		if (isNullish(input)) {
```

Comment:
not a call to action, just saying if interesting: input cannot be undefined here